src/ck-core/ck.C

```diff
@@ -2087,6 +2087,24 @@ void CkSendMsgNodeBranch(int eIdx, void *msg, int node, CkGroupID gID, int opts)
   CkpvAccess(_coreState)->create();
 }
 
+void CkSendMsgNodeBranchPe(int eIdx, void *msg, int pe, CkGroupID gID, int opts) {
+  // TODO ( expand support for these options at some point... )
+  CkAssertMsg(!((opts & CK_MSG_INLINE) || (opts & CK_MSG_IMMEDIATE)),
+              "pe-level inline and immediate sends to node branchs unsupported");
+  int numPes = (pe == CK_PE_ALL) ? CkNumPes() : 1;
```

Comment:
Maybe this should fail if given `CK_PE_ALL`?

Alternatively, should it send to all _PEs_? If so, this would currently break since it `...ClEnqueue` only does node broadcasts.